ChatIntegrationExamples/SendChatNotificationCard.xml

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<DMSScript options="272" xmlns="http://www.skyline.be/automation">
+	<Name>Send Chat Notification Card Example</Name>
+	<Description>Sends a notification using an Adaptive Card in a given groups chat or one-on-one chat in Microsoft Teams. Make sure to have a tenant set for your organization in the Admin App (https://admin.dataminer.services).</Description>
+	<Type>Automation</Type>
+	<Author>SKYLINE2\JordyAM</Author>
+	<CheckSets>FALSE</CheckSets>
+	<Folder>Examples/ChatOps/Teams/Integration</Folder>
+
+	<Protocols>
+	</Protocols>
+
+	<Memory>
+	</Memory>
+
+	<Parameters>
+		<ScriptParameter id="1" type="string" values="ChatsExample">
+			<Description>Chat Id</Description>
```

Comment:
Is it possible to change the parameter to "Chat ID", so that it is capitalized correctly?